### PR TITLE
Compare timetables: timetable colors

### PR DESCRIPTION
**Pull request by JiaqiWang18**
## Description
Work on rendering the slot colors of compare timetables.
image

Currently:
* sections in both timetables: turquoise
* sections in active timetable: red
* sections in compared timetable: blue

### CODE REVIEW

**Comment by JiaqiWang18** [comment]
this can be merged

**Comment by JJamesWWang** [comment]
ooh, looks nice. I'll review this later

**Comment by JJamesWWang** [comment]
Actually, you can merge and I can check it out later too if you want (don't wait on my review)

**Comment by JiaqiWang18** [comment]
It's fine, I don't have time to work on it today and tmrw lol

File: static/js/redux/ui/CompareTimetableSidebar.tsx [hunk 1]
```diff
@@ -38,8 +44,42 @@ const CompareTimetableSideBar = () => {
...
+        inTimetable = true;
+      }
+    });
+    return inTimetable;
+  };
+
+  /**
+   * returns an array of course ids of the courses that have the same section in both timetables
+   */
+  const getSectionsInBothTimetables = () => {
```
Comment (JJamesWWang) [inline static/js/redux/ui/CompareTimetableSidebar.tsx:51]: Do you think you could extract the relevant methods in both CompareTimetableSidebar and SlotManager into a single file (e.g. timetablesUtils or in slotUtils)? They are both dependent on the same code, and we don't want to duplicate it (as we might end up change code in one of the files but not the other, leading to strange bugs).

**Review by JJamesWWang — CHANGES_REQUESTED** [review]
Looks good, can you just refactor the duplicate code?

File: static/js/redux/ui/CompareTimetableSidebar.tsx (hunk 1 above)
Comment (JiaqiWang18) [reply static/js/redux/ui/CompareTimetableSidebar.tsx:51]: ok I will look into it

**Review by JJamesWWang — APPROVED** [review]

**Pull request merged**